index.html

```diff
@@ -14,47 +14,58 @@
 </head>
 
 <body>
-    <header>
+    <header class="flex">
         <nav class="nav-bar flex">
             <div class="nav-header flex">
-                <a class="logo flex decoration" href="/my-logo">My Logo</a>
-                <a href="/#"><i class="fa fa-bars flex"></i></a>
+                <div class="flex">
+                    <a class="logo flex decoration" href="/my-logo">My Logo</a>
+                </div>
+                <div class="nav-list flex">
+                    <a href="/#"><i class="fa fa-bars " id="menu-button"></i></a>
+                    <ul class="hidden right">
+                        <li class="item-navigation list-style"><a class="decoration" href="#">Portfolio</a></li>
+                        <li class="item-navigation list-style"><a class="decoration" href="#">About</a></li>
+                        <li class="item-navigation list-style"><a class="decoration" href="#">Contact</a></li>
+                    </ul>
+                </div>
```

Comment:
- [ ] As requested by the previous reviewer, Please fix the right margin for the header and make sure it matches the design on the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). I think it's visible fine on your machine because of the screens difference, Make sure your design matches the Figma template on all screens over 768px.
 - As suggested by the previous reviewer, You should favor units that are percent base (vh, vw, %) over fixed units (px).
 
### Your design

![image](https://user-images.githubusercontent.com/83803414/159943087-26e04702-445a-4a43-9348-418b9c731b70.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/159943234-0a02a1af-8584-488f-b5eb-bf101b14bf60.png)